tests/unit/test_charm.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Unit test for the charm."""
+
+from unittest.mock import MagicMock
+
+import ops
+import pytest
+from ops.testing import Harness
+
+import state
+from charm import CACHE_CONFIG_INTEGRATION_NAME, ContentCacheBackendsConfigCharm
+
+SAMPLE_CONFIG = {
+    state.HOSTNAME_CONFIG_NAME: "example.com",
+    state.PATH_CONFIG_NAME: "/",
+    state.BACKENDS_CONFIG_NAME: "10.10.1.1,10.1.1.2",
+    state.PROTOCOL_CONFIG_NAME: "https",
+}
+
+
+def test_start(charm: ContentCacheBackendsConfigCharm):
+    """
+    arrange: A working charm.
+    act: The charm started.
+    assert: Charm in block state.
+    """
+    assert charm.unit.status == ops.BlockedStatus("Waiting for integration")
+
+
+def test_config_no_integration(charm: ContentCacheBackendsConfigCharm, harness: Harness):
+    """
+    arrange: Charm with no integration.
+    act: Update the configuration with valid values.
+    assert: The charm in active status.
+    """
+    harness.update_config(SAMPLE_CONFIG)
+    assert charm.unit.status == ops.BlockedStatus("Waiting for integration")
+
+
+@pytest.mark.parametrize(
+    "event",
+    [
+        pytest.param("_on_config_changed", id="config_changed"),
```

Comment:
Isn't that test similar to the previous one ?

Reply:
The previous test has valid configuration.
This test does not have valid configuration.